share/man/man4/uipaq.4

```diff
@@ -55,17 +55,27 @@ The
 .Nm
 driver supports the following adapters:
```

Comment:
This line is the bigger issue for the hardware release notes.

Reply:
I noticed both uipaq(4) man pages from OpenBSD and NetBSD included DESCRIPTION sections. Should I derived from theirs to be added into ours?

Reply:
Also, it seems like this list is more device-oriented. Should they be called "devices" instead of "adapters"?

Reply:
I think that sounds nice.